### ISSUE TITLE
VS does not open when using FireFox

### ISSUE THREAD

**Issue by anddrzejb** [jsakamoto/FindRazorSourceFile#1]
Hi!

This project looks very cool & promising! 

I mostly am already working with net6 preview. When are you planning to add net6? 

Also, just for the fun of it I created new project from basic server-side template (net5). All is working great, however when in firefox the VS does not seem to open the razor files. It works fine in chrome.

Example:
![findRazor](https://user-images.githubusercontent.com/6518006/132685578-1ee50188-1c77-4964-be28-6559ca152fe5.gif)

**Comment by jsakamoto** [comment]
@anddrzejb 

> When are you planning to add net6?

Did you try it?  
In my test case, this package & Visual Studio extension worked fine on .NET6 Preview 7 & Visual Studio 2022 Preview 3.

https://twitter.com/jsakamoto/status/1432689831503548417

If you tried it and it didn't work well, I welcome your report. 😊

**Comment by jsakamoto** [comment]
@anddrzejb 

> however when in firefox the VS does not seem to open the razor files.

Thanks for report!

I'll investigate this problem.

**Comment by anddrzejb** [comment]
> > When are you planning to add net6?
> 
> Did you try it?
> In my test case, this package & Visual Studio extension worked fine on .NET6 Preview 7 & Visual Studio 2022 Preview 3.

I am still in preview 6 working  on VS2019 Preview. When I installed VS2022 (with net6 perview 7) my whole system blew-up. I even reinstalled Windows, but it did not help (I am simplyfing here, I did a loooot of steps - took me over a week of attempts). I am waiting now for preview 8, hopefully my issues will be gone by then. 

I based this issue on [this line](https://github.com/jsakamoto/FindRazorSourceFile/blob/abd12481d14044e9504c4c29e1f45d1ede1ba1a9/FindRazorSourceFile.Server/FindRazorSourceFile.Server.csproj#L4) and the fact that in VS2019 Preview & net6 preview-6 the package seems not only not working but even causes the basic project from template to fail to start. I can try to give you more details (errors on failure), if you are still interested. It might be that something is messed-up with my setup, but at this point I cannot figure out what.

**Comment by jsakamoto** [comment]
@anddrzejb 

> however when in firefox the VS does not seem to open the razor files.

Could you try to set `true` to `security.enterprise_roots.enabled` in `about:config` on Firefox?

![image](https://user-images.githubusercontent.com/95908/132856960-e8704a2b-1b94-4ad9-bf30-61c2464526e4.png)

(See also: https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-5.0&tabs=visual-studio#configure-trust-of-https-certificate-using-firefox-browser )

**Comment by anddrzejb** [comment]
Ok, it worked. I think switching `security.enterprise_roots.enabled` wasn't enough. I restarted Firefox afterwards and it still did not work. Then I added `profile.json` file (which as far as I understand, should already be there, but wasn't), restarted FF and that seemed to do the trick. Thanks! I changed the title of the issue and closed it. If I solve my net6 issues but continue to have a problem with your package, I will open a new one. Thanks!
